pkgs/tools/filesystems/darling-dmg/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, lib, cmake, fuse, zlib, bzip2, openssl, libxml2, icu } :
+
+stdenv.mkDerivation rec {
+  name = "darling-dmg-${version}";
+  version = "1.0";
+  src = fetchFromGitHub {
+    owner = "darlinghq";
+    repo = "darling-dmg";
+    rev = "9522e3907b82f6cde141b3e0e0063f09c8710cbf";
```

Comment:
After looking at the upstream repo: There are tags available which should be used IMHO. This rev refers to version 1.0.3, not 1.0. In the meantime 1.0.4 was released btw. :)